src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -1225,12 +1222,9 @@
   "Given a path to a custom OID mappings file, return a map of all oids to
   shortnames"
   [custom-oid-mapping-file :- (schema/maybe schema/Str)]
```

Comment:
It seems that `schema/maybe` is unnecessary here since Puppet has a default value for the setting.